virtualizarr/xarray.py

```diff
@@ -397,6 +412,18 @@ def to_kerchunk(
 
             return None
         elif format == "parquet":
-            raise NotImplementedError()
+            from kerchunk.df import refs_to_dataframe
+
+            if isinstance(filepath, Path):
+                url = str(filepath)
+            elif isinstance(filepath, str):
+                url = filepath
+
+            return refs_to_dataframe(
+                refs,
```

Comment:
Where is this `refs` argument coming from? It's not an argument to the method...

Reply:
It is defined in line 402 before the if/else block starts.

Reply:
Sorry yes I just saw that - stupid comment sorry 😅